lib/resources/r4/observation.yaml

```diff
@@ -414,13 +414,39 @@ fields:
                 "text": "primary performer"
              },
              "url": "http://hl7.org/fhir/StructureDefinition/event-performerFunction"
+          },
+          {
+             "valueCodeableConcept":{
+                "coding":[
+                   {
+                      "system":"http://terminology.hl7.org/CodeSystem/v3-ParticipationType",
+                      "code":"PRF",
+                      "display":"performer"
+                   }
+                ],
+                "text":"performer"
+             },
+             "url":"http://hl7.org/fhir/StructureDefinition/event-performerFunction"
+          },
+          {
+             "valueCodeableConcept":{
+                "coding":[
+                   {
+                      "system":"http://terminology.hl7.org/CodeSystem/v3-ParticipationType",
+                      "code":"AUTHEN",
+                      "display":"authenticator"
+                   }
+                ],
+                "text":"authenticator"
+             },
+             "url":"http://hl7.org/fhir/StructureDefinition/event-performerFunction"
           }
        ]
     }
   note: |
     <ul>
-       <li> Vital Signs supports only `LA` extension. </li>
-       <li> Laboratory supports `LA` , `PPRF` and `OP` extensions. </li>
+       <li> Vital Signs supports `LA` and `PPRF` extension. The code must be 'LA' when only one performer is provided and when multiple performers are present then the code must be 'LA' for one of the performers.</li>
```

Comment:
I like the new information you've added. I'm wondering if it might be more-clear as sub-bullets:

> * Vital Signs supports `LA` and `PPRF` extensions.
>   * When only one performer is provided, the extension must be `LA`.
>   * When multiple performers are provided, the the code must be `LA` for one of the performers.

Reply:
Instead of 'extension', I think we should say 'codes' or 'code' as appropriate.

Reply:
<img width="971" alt="Screen Shot 2021-06-17 at 2 48 38 PM" src="https://user-images.githubusercontent.com/18075594/122463387-2befe080-cf7b-11eb-830a-769495d14727.png">